bindings/wasm/package.json

```diff
@@ -16,17 +16,14 @@
     "build:nodejs": "npm run build:src && npm run bundle:nodejs && wasm-opt -O node/identity_wasm_bg.wasm -o node/identity_wasm_bg.wasm",
     "build:web": "npm run build:src && npm run bundle:web && wasm-opt -O web/identity_wasm_bg.wasm -o web/identity_wasm_bg.wasm",
     "build:docs": "node ./build/docs",
-    "build:examples": "webpack --config ./examples/webpack.config.js --mode=production",
     "build": "npm run build:web && npm run build:nodejs && npm run build:docs",
-    "example:node": "node examples/dist/node.js",
     "example:browser": "http-server ./examples/dist/ -c-1 -o ",
     "example:account": "ts-node ./examples-account/src/node.ts",
-    "example:stardust": "ts-node ./examples-stardust/src/main.ts",
+    "example:node": "ts-node ./examples/src/main.ts",
     "test": "npm run test:unit && npm run test:unit:node && npm run test:examples",
-    "test:examples": "concurrently -g --timings \"npm:test:node\" \"npm:test:account:node\" \"npm:test:stardust\" \"npm:test:browser:parallel\" \"npm:test:readme\"",
-    "test:node": "mocha ./examples/dist/tests/node/*.js --parallel --jobs 4 --retries 3 --timeout 180000 --exit",
+    "test:examples": "concurrently -g --timings \"npm:test:node\" \"npm:test:readme\"",
     "test:account:node": "ts-mocha -p tsconfig.node.json ./examples-account/src/tests/*.ts --parallel --jobs 4 --retries 3 --timeout 180000 --exit",
-    "test:stardust": "ts-mocha -p tsconfig.node.json ./examples-stardust/src/tests/*.ts --parallel --jobs 4 --retries 3 --timeout 180000 --exit",
+    "test:node": "ts-mocha -p tsconfig.node.json ./examples/src/tests/*.ts --parallel --jobs 4 --retries 3 --timeout 180000 --exit",
     "test:browser:parallel": "cypress-parallel -s test:browser -t 4 -d cypress/e2e -a '\"--quiet\"'",
     "test:browser": "cypress run --headless",
     "test:readme": "mocha ./tests/txm_readme.js --retries 3 --timeout 180000 --exit",
```

Comment:
I would like to keep the setup and get a smoke test in cypress running ASAP. I will look into this, but not sure it can be done before we want to merge this branch,